core/src/main/java/com/hcl/ecomm/core/config/MagentoServiceConfig.java

```diff
@@ -24,6 +24,13 @@
   String customerSignupPath = "/V1/customers";
   String customerSigninPath = "/V1/integration/customer/token";
   String customerProfilePath = "/V1/customers/me";
+    String customerCreateCartPath = "/V1/carts/mine";
```

Comment:
@DeepaliAgrwl : is there no runmode configuration for this OSGI config

Reply:
No @rishabh-behl . Right now we don't have any runmode configuration for this magentoServiceConfigs as we discussed that we will have that once we have our dedicated environment ready